drivers/include/nrfx_nvmc.h

```diff
@@ -261,6 +261,20 @@ void nrfx_nvmc_words_write(uint32_t address, void const * src, uint32_t num_word
  */
 uint16_t nrfx_nvmc_otp_halfword_read(uint32_t address);
 
+/**
+ * @brief Function for reading a 32-bit aligned word from the UICR
+ *
+ * This function should be used to read from the UICR since reading
+ * the flash main memory area straight after reading the UICR results
+ * in undefined behaviour for nRF9160.
+ * nRF9160_Rev_2_Errata_v1.0_PAN_7.
```

Comment:
Other revisions are also affected, so maybe just something like "(see anomaly 7)" (the term PAN should not be used) would suffice here.